Dockerfile_release

```diff
@@ -0,0 +1,3 @@
+FROM scratch
+COPY driftctl /usr/bin/driftctl
+ENTRYPOINT ["/usr/bin/driftctl"]
```

Comment:
What's the purpose of this dockerfile ?

Reply:
It was useful for goreleaser to build and push the Docker image, but it's not useful anymore since we decided to not use goreleaser for that.